.travis.yml

```diff
@@ -116,8 +119,7 @@ matrix:
 
         - os: linux
           stage: Initial tests
-          env: CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PYTEST_VERSION='>3.2'
```

Comment:
@bsipocz Hey, I just noticed this from a rebase - was the ``PYTEST='>3.2'`` drop intended?

Reply:
yes, we only needed it while there was a version pinning in ci-helpers, but that got removed long ago in https://github.com/astropy/ci-helpers/pull/257